src/webview/webview2/mod.rs

```diff
@@ -616,11 +616,16 @@ window.addEventListener('mousemove', (e) => window.chrome.webview.postMessage('_
           });
         }
 
-        win32wm::WM_SETFOCUS => {
+        win32wm::WM_ENTERSIZEMOVE | win32wm::WM_SETFOCUS => {
           let controller = dwrefdata as *mut ICoreWebView2Controller;
           let _ = (*controller).MoveFocus(COREWEBVIEW2_MOVE_FOCUS_REASON_PROGRAMMATIC);
         }
 
+        win32wm::WM_MOVE | win32wm::WM_MOVING => {
```

Comment:
this works and actually we can optimize this and only call it when moving/resizing is finished
```suggestion
        win32wm::EXITSIZEMOVE=> {
```

Reply:
yeah i was a little scared from their docs https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2controller.notifyparentwindowpositionchanged?view=webview2-dotnet-1.0.1343.22
> Tells WebView that the main WebView parent (or any ancestor) HWND moved.
> This is needed for accessibility and certain dialogs in WebView to work correctly.

Reply:
Found different docs where they show move and moving too: https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2host?view=webview2-0.9.430#notifyparentwindowpositionchanged

I mean, in this specific use case exitsizemove works perfect, but i wonder what else its used for. Imagine of OS dpcs would actually document stuff x)

Reply:
hmm actually I think it is better to use `WM_MOVE` and `WM_SIZE` so it would account for maximized and fullscreen changes

Reply:
My local copy still has the move + moving and resizing + maximizing seems to be handled by that already.

Reply:
`WM_MOVE` is fine but `WM_MOVING` can be triggered multiple times and so we don't need to keep calling it every time we just need to call it once when moving is done.

Reply:
`WM_WINDOWPOSCHANGED` also works

Reply:
man win32 has lots of messages for the same damn thing

Reply:
> My local copy still has the move + moving and resizing + maximizing seems to be handled by that already.

Same for fullscreen i think. it's a bit tricky to test.

> WM_MOVE is fine but WM_MOVING can be triggered multiple times

Aren't they basically the same event? As-in they both get triggered continuously while the window is dragged?

> so we don't need to keep calling it every time we just need to call it once when moving is done.

Yeah well, that depends on what Microsoft's intention behind the notify api is x)

> WM_WINDOWPOSCHANGED also works

the docs of that sounded like it's only triggered by programmatic position changes to me x)

Anyway, i'm fine with sticking to WM_EXITSIZEMOVE until we see something that needs some spammier events like the docs suggests. I can't think of anything 🤷

Reply:
> Aren't they basically the same event? As-in they both get triggered continuously while the window is dragged?

I think `WM_MOVE` only triggers at the end but I haven't tested it though but should be according to the docs.

> the docs of that sounded like it's only triggered by programmatic position changes to me x)

yeah the docs is misleading because I tested it and it seems to work, maybe test this one more time before I merge?

Reply:
@FabianLars could you test with the latest commit using `WM_WINDPOSCHANGED` before I merge?

Reply:
works fine on my end.

Reply:
cool